packages/create-svelte/templates/default/src/lib/Counter/minus.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="2" viewBox="0 0 16 2" fill="black" stroke="black" xmlns="http://www.w3.org/2000/svg">
+	<path d="M1.25 1H15.25" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round" />
+</svg>
```

Comment:
should we add a newline to the end of this file and `plus.svg`?

Reply:
ended up removing them